core/src/main/java/com/shatteredpixel/shatteredpixeldungeon/items/wands/Wand.java

```diff
@@ -695,8 +695,8 @@ public void onSelect( Integer target ) {
 							return;
 						}
 
-						float shield = curUser.HT * (0.04f*curWand.curCharges);
-						if (curUser.pointsInTalent(Talent.SHIELD_BATTERY) == 2) shield *= 1.5f;
+						float ratio = .02f + 0.015f * curUser.pointsInTalent(Talent.SHIELD_BATTERY);
+						float shield = curUser.HT * ratio;
```

Comment:
I think you forgot to scale it with current charges.
```suggestion
						float shield = curUser.HT * ratio * curWand.curCharges;
```

Reply:
fixed